src/frontend/src/apis/finance.api.ts

```diff
@@ -192,3 +192,24 @@ export const downloadBlobsToPdf = async (blobData: Blob[], filename: string) =>
   // Save the Blob as a file using file-saver
   saveAs(pdfBlob, filename);
 };
+
+/**
+ * API call to get the list of Reimbursement Requests that are pending advisor approval
+ *
+ * @returns The list of Reimbursement Requests that are pending advisor approval
+ */
+export const getPendingAdvisorList = () => {
+  return axios.get(apiUrls.financeGetPendingAdvisorList());
```

Comment:
call transformResponse with the frontend reimbursementRequestTransformer on all the returned reimbursement requests like we do for getAllReimbursementRequests